ktor-http/ktor-http-cio/common/src/io/ktor/http/cio/websocket/WebSocketExtension.kt

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.http.cio.websocket
+
+import io.ktor.util.*
+
+/**
+ * WebSocket extensions API is experimental according to [KTOR-688](https://youtrack.jetbrains.com/issue/KTOR-688)
+ * To get more information about Ktor experimental guarantees consult
+ * with [KTOR-1035](https://youtrack.jetbrains.com/issue/KTOR-1035).
+ */
+@RequiresOptIn(
+    message = "WebSocket extensions API is experimental according to []()." +
```

Comment:
Missing ticket number here and space character before the following text

Reply:
Thanks